src/test/java/com/zaiats/catcher/controller/CandidateControllerTest.java

```diff
@@ -0,0 +1,72 @@
+package com.zaiats.catcher.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.zaiats.catcher.model.CandidateModel;
+import com.zaiats.catcher.service.CandidateService;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(SpringExtension.class)
+@WebMvcTest(controllers = CandidateController.class)
+class CandidateControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @MockBean
+    private CandidateService candidateService;
+
+    @Test
+    void getAllCandidates_returns200() throws Exception {
+        CandidateModel candidateModel = new CandidateModel(
+                20230122222130L, "Alex", "Bip", "alex.bip@mail.com", "Developer");
```

Comment:
It would be better to take it outside the test because it can be useful in the following tests =)

Reply:
Sure! Done!